llpc/CMakeLists.txt

```diff
@@ -571,4 +572,3 @@ if(LLPC_BUILD_LIT)
     add_subdirectory(test)
 endif()
 endif()
-
```

Comment:
nit: unintended change?

Reply:
Yes, I did not remove this intentionally. However I think it’s cleaner to not have empty lines at the end of a file. So if nobody objects I would leave that in there. Otherwise I’m happy to split that out into a separate PR.

Reply:
Empty lines at the end of files look better in my editor, but I generally don't care.
I'd group such cosmetic changes into a separate PR if you think there's value in such a refactoring and argue for it.